WpfTest/ViewModel/MainPageViewModel.cs

```diff
@@ -0,0 +1,85 @@
+    using System.Collections.ObjectModel;
+using System.Windows.Controls;
+using System.Windows.Input;
+using WpfTest.Commands;
+using WpfTest.View;
+using WpfTest.Model;
+
+namespace WpfTest.ViewModel;
+
+public class MainPageViewModel : BaseViewModel
+    {
```

Comment:
the same here, please use dotnet-format tool to format .cs files, see https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format